mountpoint-s3-client/src/user_agent.rs

```diff
@@ -164,4 +172,12 @@ mod tests {
             "Java_HotSpot_-TM-_64-Bit_Server_VM"
         );
     }
+
+    #[test]
+    fn test_multiple_values() {
+        let mut user_agent = UserAgent::new(None);
+        user_agent.key_values("mp-cache", &["shared", "local"]);
+        let user_agent_string = user_agent.build();
+        assert!(user_agent_string.contains("md/mp-cache#shared+local"));
+    }
```

Comment:
Just acknowledging that the ordering is currently based on the slice and could change. I think that assumption is fine, any monitoring logic should assume it could change.